src/Tizen.NUI.Scene3D/src/internal/CameraBindableProperty.cs

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.ComponentModel;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI.Scene3D
+{
+    public partial class Camera
```

Comment:
please add "[EditorBrowsable(EditorBrowsableState.Never)]" on Camera class also.

Reply:
To add "[EditorBrowsable(EditorBrowsableState.Never)]" on both of bindableproperty and class makes build error.